src/utils.jl

```diff
@@ -1834,7 +1834,18 @@ function _rand_invertible_matrix(F::CTFieldTypes, n::Integer)
     return A
 end
 
-
+function _rand_gf_elem(field, rng::AbstractRNG = Random.seed!())
```

Comment:
I'm too lazy to look, but if this is not in the section of the file marked finite fields, please move it there